src/styles/theme.js

```diff
@@ -5,6 +5,7 @@ import 'react-toastify/dist/ReactToastify.css';
 const theme = createTheme({
   palette: {
     primary: blue,
+    link: '#2196f3',
```

Comment:
Material UI already has a link component. Please remove that and use [this](https://v4.mui.com/pt/components/links/#links).